README.md

```diff
@@ -18,26 +18,27 @@ almost any database system; So far it's been used with SQLite, Postgres, and Mic
 Just specify a database url on the command line with `-d`.
 
 ### Example Use
+Installation via `pip install -e .`
 
 1. **Bay Area Rapid Transit with GTFS-realtime TripUpdates:**
 
    a. Using SQLite:
 
-       gtfsrdb.py -t http://api.bart.gov/gtfsrt/tripupdate.aspx -d sqlite:///test.db -c
+       gtfsrdb -t http://api.bart.gov/gtfsrt/tripupdate.aspx -d sqlite:///test.db -c
```

Comment:
URL throws 403, maybe use http://api.tampa.onebusaway.org:8088/trip-updates for testing